packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -0,0 +1,361 @@
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { isPhone } from "@ui5/webcomponents-base/dist/Device.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import Dialog from "@ui5/webcomponents/Dialog.js";
+import Button from "@ui5/webcomponents/Button.js";
+import Bar from "./Bar.js";
+
+import {
+	VSD_DIALOG_TITLE_SORT,
+	VSD_SUBMIT_BUTTON,
+	VSD_CANCEL_BUTTON,
+	VSD_RESET_BUTTON,
+	VSD_SORT_ORDER,
+	VSD_SORT_BY,
+	VSD_ORDER_ASCENDING,
+	VSD_ORDER_DESCENDING,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Template
+import ViewSettingsDialogTemplate from "./generated/templates/ViewSettingsDialogTemplate.lit.js";
+
+// Styles
+import viewSettingsDialogCSS from "./generated/themes/ViewSettingsDialog.css.js";
+
+const metadata = {
+	tag: "ui5-view-settings-dialog",
+	managedSlots: true,
+	properties: /** @lends  sap.ui.webcomponents.fiori.ViewSettingsDialog.prototype */ {
+		/**
+		 * Defines the initial sort order.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		 sortDescending: {
+			type: Boolean,
+		},
+		/**
+		 * Keeps recently focused list in order to focus it on next dialog open.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _recentlyFocused: {
+			type: Object,
+
+		},
+		/**
+		 * Stores settings of the dialog before the first open.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _initialSettings: {
+			type: Object,
+		},
+		/**
+		 * Stores settings of the dialog after confirmation.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _confirmedSettings: {
+			type: Object,
+		},
+		/**
+		 * Stores current settings of the dialog.
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		 _currentSettings: {
+			type: Object,
+		},
+	},
+	slots: /** @lends  sap.ui.webcomponents.fiori.ViewSettingsDialog.prototype */ {
+		/**
+		 * Defines the <code>sorting</code> list.
+		 * @type {sap.ui.webcomponents.fiori.List}
+		 * @slot sorting
+		 * @public
+		 */
+		 "sorting": {
```

Comment:
When I read "sorting" I could not understand what is expected to be passed.
Then I can see @type {sap.ui.webcomponents.fiori.List} and in the html page
I can see that list items are provied ui5-li slot="sorting">Name</ui5-li

Isn't sortItems more accurate or there is other reason to name it "sorting"?

Reply:
ummm, it was sortBy, but then Filip said that he prefer something with one word without capitals... see jhis comments on first review...

Reply:
Done